lib/Service/DocumentService.php

```diff
@@ -265,7 +265,7 @@ public function addStep(Document $document, Session $session, array $steps, int
 
 		return [
 			'steps' => $stepsToReturn,
-			'version' => $newVersion,
+			'version' => isset($documentState) ? $document->getLastSavedVersion() : 0,
```

Comment:
Can you tell if this has any other side effects? We now no longer return the latest version (number of steps in the db) but just the last saved. We used this in the past to compare and see if the document had unsaved changes, but I think that part was changed already.

Reply:
Otherwise `$newVersion` is no longer used as a variable

Reply:
Good catch wrt `$newVersion`. We were not using the version in the response on the client side anymore.

The unsaved changes check is on the server when cleaning up docs, right? That should still work. this is not changing the data structures there.